<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>场集工具</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>服务器路径</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="serverEdit">
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="serverButton">
           <property name="text">
            <string>浏览</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QLineEdit" name="videoFolderEdit"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>服务器视频路径</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="imageFolderEdit"/>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>服务器单帧文件夹</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Nuke路径</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="nukeEdit">
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="nukeButton">
           <property name="text">
            <string>浏览</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>工作目录</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="dirEdit">
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="dirButton">
           <property name="text">
            <string>浏览</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>工程名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="projectEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="epEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>集</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="scEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>场</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>视频上传至</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="videoDestEdit">
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="imageDestEdit">
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>单帧上传至</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="csheetNameEdit">
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>色板名称</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>色板上传至</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="csheetDestEdit">
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QFormLayout" name="formLayout_4">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>色板素材文件夹</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="csheetFFNameEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>色板名称前缀</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="csheetPrefixEdit"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>色板底板</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="backDropBox"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="csheetUpCheck">
           <property name="text">
            <string>创建后上传色板</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="csheetOpenCheck">
           <property name="text">
            <string>创建后打开色板</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sheetButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>创建色板</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="openCSheetButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>打开色板</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="1">
            <widget class="QLineEdit" name="imageFNameEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="videoFNameEdit"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>视频文件夹名称</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>单帧文件夹名称</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>上传</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QCheckBox" name="videoUpCheck">
             <property name="text">
              <string>视频</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="imageUpCheck">
             <property name="text">
              <string>单帧</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>下载</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QCheckBox" name="videoDownCheck">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>视频</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="imageDownCheck">
             <property name="text">
              <string>单帧</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="syncButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>上传及下载</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <widget class="QLabel" name="version_label">
     <property name="text">
      <string>version</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
